ga4gh/frontend.py

```diff
@@ -13,11 +13,14 @@
 import flask.ext.cors as cors
 
 app = api.FlaskAPI(__name__)
-app.config.from_object('ga4gh.server.config:DefaultConfig')
-if os.environ.get('GA4GH_CONFIGURATION') is not None:
-    app.config.from_envvar('GA4GH_CONFIGURATION')
 
-cors.CORS(app, allow_headers='Content-Type')
+
+def configure(args):
```

Comment:
If a config can not be found the app.config.from_object(configStr) will fail with "ImportError: No module named DummyConfig," so implicitly that's already taken care of.